### PR TITLE
Maximum GCD Editorial

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
i swear i wrote a sol for something involving max gcd a couple months ago

**Review by SansPapyrus683 — APPROVED** [review]
you're my little pogchamp

File: solutions/gold/cc-KSIZEGCD.mdx [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+id: cc-KSIZEGCD
```
Comment (SansPapyrus683) [inline solutions/gold/cc-KSIZEGCD.mdx:2]: make change the problem id to be lowercase

File: solutions/gold/cc-KSIZEGCD.mdx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+</Spoiler>
+
+<Spoiler title="Solution">
+
+## Explanation
+
+Since we know that the larger the subarray is, the GCD will never increase, we can create a list of hashmaps where
+$\texttt{max\_size}_{i, \text{gcd}} =$ the maximum size of the subarray ending at $i$ with a GCD of $\text{gcd}$.
+
+Then, we can perform updates in $\mathcal{O}(\log N)$ by extending subarrays at $i - 1$.
```
Comment (envyaims) [inline solutions/gold/cc-KSIZEGCD.mdx:29]:
```suggestion
Then, we can perform updates in $\mathcal{O}(\log N)$ by extending subarrays that ended at index $i - 1$ to index $i$.
```

**Review by envyaims — APPROVED** [review]
slay

File: solutions/gold/cc-KSIZEGCD.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/gold/cc-KSIZEGCD.mdx]: bump

**Comment by SansPapyrus683** [comment]
my name is walter hartwell white. i live at 308 negra arroyo lane, albuquerque, new mexico, 87104.

**Pull request merged**